### DISCUSSION TITLE
add a feature to turn off copilot by default

### DISCUSSION THREAD

**Discussion by kylrth** [community/community#46973] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi, I don't want to be bothered about logging into GitHub every time I open VS Code.

### current experience

Any time I open VS Code, it asks me to sign in to GitHub. When I close that notification, another one pops up saying that copilot could not connect to GitHub because I did not log in.

### desired experience

If I want to use copilot during a particular coding session, I want to click the status bar icon, log into GitHub, and then use it. Otherwise, I don't want to be bothered.

**Comment by tilmann1** [comment]
@kylrth Can you please update to version 1.72.8614 of the VS Code extension and let us know whether you're still seeing the same symptoms? Thanks!

**Reply by kylrth** [reply]
Yes, I see this on 1.72.8614.
